specification/vision/ImageAnalysis/routes.tsp

```diff
@@ -0,0 +1,114 @@
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@typespec/http";
+
+import "./models.tsp";
+
+using Azure.Core.Traits;
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+
+namespace ImageAnalysis;
+
+#suppress "@azure-tools/typespec-azure-core/byos" "It's an RPC call to analyze an image, it doesn't store anything. There's a BYOS version of this call, but it's not this one."
+@doc("Performs a single Image Analysis operation")
+@actionSeparator("/")
```

Comment:
Remove the actionSeparator, change the route to be only `imageanalysis`

Reply:
Actually, need to look more at that. Something is broken here. And it isn't (only) the spec.

Reply:
Remove action and action separator is the better recommendation :).

Reply:
Done! removed all @action and @actionSeparator. Everything works fine without it.